### ISSUE TITLE
Bump up to Drush 10 from Drush 8

### ISSUE THREAD

**Issue by dsteplight** [aegir-project/dockerfiles#49]
It looks like most of these Docker files are using Drush 8. Can we have it bumped up to using Drush 10?

**Comment by PaulLebmann** [comment]
I don't think that this is going to happen. Have a look at: [https://www.drupal.org/project/provision/issues/2953349](https://www.drupal.org/project/provision/issues/2953349)
